bin/clean_test.js

```diff
@@ -1,28 +0,0 @@
-/*
-       Licensed to the Apache Software Foundation (ASF) under one
-       or more contributor license agreements.  See the NOTICE file
-       distributed with this work for additional information
-       regarding copyright ownership.  The ASF licenses this file
-       to you under the Apache License, Version 2.0 (the
-       "License"); you may not use this file except in compliance
-       with the License.  You may obtain a copy of the License at
-
-         http://www.apache.org/licenses/LICENSE-2.0
-
-       Unless required by applicable law or agreed to in writing,
-       software distributed under the License is distributed on an
-       "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-       KIND, either express or implied.  See the License for the
-       specific language governing permissions and limitations
-       under the License.
-*/
-
-const fs = require('fs');
-
-if (fs.existsSync('./test/gradlew')) {
```

Comment:
This was added to easily clean gradle wrappers inside the tests, although I think the paths needs to be updated to `test/android` and `test/androidx`.

I think I'd prefer to keep this script and update the paths. The intent was to clean up gradle wrappers so that when you upgrade the gradle version, the new version will be installed. If gradle is already present, it won't install the newer declared version.

Reply:
I wanted to mention `git clean`, but I saw that this was already discussed when this was added. I will remove this file's deletion from this PR.